ci/deps/dask.yml

```diff
@@ -0,0 +1 @@
+dask
```

Comment:
do we need to list distributed at all? (even if for testing)?

Reply:
Hmmm - good question. I wasn't using it for any of my tests previously (just relying on the default scheduler), but I think it is worth adding in the future. 

Would the expectation be that users pass in the `dask.distributed.Client` to the ibis client?

Reply:
yeah we would want to pass this to the ibis client i think.

Reply:
I've added this to the TODO list in https://github.com/ibis-project/ibis/issues/2537. I'll update the requirements for distributed when I add that.